orchagent/portsorch.cpp

```diff
@@ -2838,8 +2838,6 @@ void PortsOrch::doLagTask(Consumer &consumer)
                         continue;
                     }
 
-                    gNeighOrch->ifChangeInformNextHop(alias,
-                                                 (operation_status == "up"));
```

Comment:
can you explain why are you moving this function to line 2875?

Reply:
My thinking is if lag Port object does not exist, there should be no router interface created on lag. If lag router interface does not exist, there should be no next hop object created upon lag. `ifChangeInformNextHop` call will have void operation in such a case.

`ifChangeInformNextHop` call is meaningful if at least lag Port object exists. This is also true for sub interfaces created on Lag, because they are kept in m_child_ports data member of parent lag's Port object.

This is also my question when I studied #1030, why it is not deferred to the field value processing block starting on line 2869? Would like to know the idea behind it.

Reply:
i am ok for this move. but i think it is better to refactor the code to move the block 2873-2885 into updatePortOperStatus(). Essentially, they are the same code, we should reuse the code there.

Reply:
can you make the above change?